docs/Personas.md

```diff
@@ -0,0 +1,9 @@
+### Personas
+
+- Eugenio Trujillo, cacereño, 35 años, programador y escalador aficionado con más de 15 años de
```

Comment:
De esta persona entiendo la "experiencia". ¿Es lo que habéis tenido en cuenta en sus HUs?

Reply:
Resuelto modificando historias de usuario